JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/slurm/job/JobSubmission.java

```diff
@@ -439,6 +443,17 @@ public void monitorJobs() throws SlurmJobException{
 				session = jsch.getSession(slurmJobProperty.getHpcServerLoginUserName(), getHpcAddress(), 22);
 				String pwd = slurmJobProperty.getHpcServerLoginUserPassword();
 				session.setPassword(pwd);
+
+				try {
+					Connector con = new PageantConnector();
+					IdentityRepository irepo = new RemoteIdentityRepository(con);
+					jsch.setIdentityRepository(irepo);
+					session.setConfig("PreferredAuthentications", "publickey,keyboard-interactive,password");
+				} catch (AgentProxyException e) {
+					session.setConfig("PreferredAuthentications", "password");
+					logger.info("Failed to load identities from Pageant", e);
```

Comment:
Avoiding excessive logging is one of the coding guidelines for JPS. https://trello.com/c/jIfJbIdc/760-create-a-list-of-peer-review-criteria-for-codes-before-merging-into-develop-branch (this will be available soon on github for the reference)

Reply:
I had not been made aware of this document so thank you for pointing it out to me. I will remove this logging statement.

Reply:
You're welcome.